common_functions/work_with_document.py

```diff
@@ -0,0 +1,20 @@
+import os
+import csv
+
+
+def create_directory():
+    '''function creates a directory to save files'''
+    try:
+        os.mkdir(os.path.join(os.environ['PYTHONPATH'], 'resulting data'))
```

Comment:
We cannot rely on `os.environ['PYTHONPATH']`, for example. here is my value of PYTHONPATH: 
```
/home/yauheni/work/bigdatalab/2020/Tasenga/ITechArt_Android_App_Review:/home/yauheni/.IntelliJIdea2019.3/config/plugins/python/helpers/pycharm_matplotlib_backend:/home/yauheni/.IntelliJIdea2019.3/config/plugins/python/helpers/pycharm_display:/home/yauheni/.IntelliJIdea2019.3/config/plugins/python/helpers/third_party/thriftpy:/home/yauheni/.IntelliJIdea2019.3/config/plugins/python/helpers/pydev:/home/yauheni/.IntelliJIdea2019.3/system/cythonExtensions:/home/yauheni/work/bigdatalab/2020/Tasenga/ITechArt_Android_App_Review/analysis_of_products_in_stores
```

Reply:
Just let the user specify where to store results, it's much esier

Reply:
+ Let's start using https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir, there is exists_ok flag, then you don't need to use try-except

Reply:
Done and I added possibility to not enter the path, in this case base_path will be like path of running file

`print('Enter path to directory to save results')
    user_path = input()
    if user_path == '':
        path = dirname(abspath(__file__))
    else:
        path = user_path`

`def save_file(path, name, data, mode='w'):
    '''function creates a file from the transferred data'''
    
    Path(join(path, 'resulting data')).mkdir(parents=True, exist_ok=True)
    with open(join(path, 'resulting data', name), mode, newline='') as csv_file:`

Reply:
It can be even simple:
```
    out_dir = Path(join(path, "resulting data"))
    out_dir.mkdir(parents=True, exist_ok=True)
    with Path(out_dir, name).open(mode=mode, newline="") as csv_file:
        writer = csv.writer(csv_file, delimiter="\t")
        for line in data:
            writer.writerow(line)
        writer.writerow("")
```

Please, check [doc](https://docs.python.org/3/library/pathlib.html#pathlib.Path.open)

Reply:
I see. Done.